### PR TITLE
Add secretMounts for the coordinator and worker

### PR DESCRIPTION
**Pull request by hagaibarel**
Supersedes and closes #79 and #46 as well, see there for additional details and discussions

### CODE REVIEW

**Comment by hagaibarel** [comment]
@mosabua would love a review here, I know it's a feature a lot of users are waiting for (us included)

File: charts/trino/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.12.0
+version: 0.13.0
```
Comment (mosabua) [inline charts/trino/Chart.yaml:18]: Remove the version update since it cause an automatic release

**Comment by mosabua** [comment]
I think this all looks good. Wdyt @losipiuk

**Review by mosabua — CHANGES_REQUESTED** [review]
Need to get the version out but otherwise lgtm

File: charts/trino/Chart.yaml (hunk 1 above)
Comment (hagaibarel) [reply charts/trino/Chart.yaml]: Done in https://github.com/trinodb/charts/pull/98/commits/de2465df0d928cd2580101878c1657be359a1807

**Review by losipiuk — COMMENTED** [review]
lgtm - but there are some linter errors

**Comment by hagaibarel** [comment]
All the checks look green, not sure what linting errors are you referring to

**Comment by losipiuk** [comment]
> All the checks look green, not sure what linting errors are you referring to

Looks fine now. Not sure what I saw back then

**Review by losipiuk — APPROVED** [review]

**Comment by losipiuk** [comment]
Please squash the commits to one.

**Review by mosabua — APPROVED** [review]
Looks good. I am going ahead with merging (and squashing)

**Comment by hagaibarel** [comment]
Thanks a lot folks!

**Pull request merged**